app/main.py

```diff
@@ -1 +1,41 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return f"{{Name: {self.name}, " \
+               f"Health: {self.health}, " \
+               f"Hidden: {self.hidden}}}"
+
+    def set_health(self, health: int) -> None:
+        self.health = health
+
+    def is_hidden(self) -> bool:
```

Comment:
Where is this method used?

Reply:
fixed

Reply:
I'd rather remove this method, as I don't think it adds any functionality, but that's just my opinion)

Reply:
no need to create this method, u can just call an attribute